portlets/src/main/frontend/src/apps/profileStats/components/GamificationRank.vue

```diff
@@ -68,9 +68,9 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
               :username="leaderBoardArray[1].remoteId"
               :title="leaderBoardArray[1].fullname"
               :avatar-url="leaderBoardArray[1].avatarUrl"
-              :size="35"
+              :size="40"
               :retrieve-extra-information="false"
-              class="me-1 pa-0 mt-0 mb-1 rounded-circle elevation-1"/>
+              class="me-2 ml-2 pa-0 mt-0 mb-1 rounded-circle elevation-1"/>
```

Comment:
How does this work for RTL ?

Reply:
The two properties are applied to two different tags, (the rounded-circle and elevation-1) created an element under the avatar to show a shadow but was,n't correctly positioned, so i applied a margin for it, and this still working fine in RTL